packages/react-components/react-list-preview/stories/List/ListActiveElement.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Button,
+  makeStyles,
+  Persona,
+  shorthands,
+  mergeClasses,
+  Text,
+  tokens,
+  SelectionItemId,
+} from '@fluentui/react-components';
+import { Mic16Regular } from '@fluentui/react-icons';
+import { List, ListItem } from '@fluentui/react-list-preview';
+
+import * as React from 'react';
+const names = [
+  'Melda Bevel',
+  'Demetra Manwaring',
+  'Eusebia Stufflebeam',
+  'Israel Rabin',
+  'Bart Merrill',
+  'Sonya Farner',
+  'Kristan Cable',
+];
+
+type Item = {
+  name: string;
+  id: string;
+  avatar: string;
+};
+
+const items: Item[] = names.map(name => ({
+  name,
+  id: name,
+  avatar:
+    'https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-react-assets/persona-male.png',
+}));
```

Comment:
This can be simplified

```suggestion
const items = [
  'Melda Bevel',
  'Demetra Manwaring',
  'Eusebia Stufflebeam',
  'Israel Rabin',
  'Bart Merrill',
  'Sonya Farner',
].map(name => ({
  name,
  id: name,
  avatar:
    'https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-react-assets/persona-male.png',
}));
```